source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -93,6 +93,80 @@ The operations are categorized by purpose and functionality.
    from the Query API expression provided in each example. However,
    both expressions will produce the same aggregation result.
 
+Arithmetic Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can use arithmetic operations to manipulate numbers. You can perform an
+arithmetic operation on a value of ``MqlInteger`` or ``MqlNumber`` using the
+methods described in this section.
+
+The following table lists the arithemetic methods
+available in the driver and corresponding expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `abs() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#abs()>`__
+     - :manual:`$abs </reference/operator/aggregation/abs/>`
+
+   * - `add() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#add(com.mongodb.client.model.mql.MqlNumber)>`__
+     - :manual:`$add </reference/operator/aggregation/add/>`
+
+   * - `divide() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#divide(com.mongodb.client.model.mql.MqlNumber)>`__
+     - :manual:`$divide </reference/operator/aggregation/divide/>`
+
+   * - `multiply() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#multiply(com.mongodb.client.model.mql.MqlNumber)>`__
+     - :manual:`$multiply </reference/operator/aggregation/multiply/>`
+
+   * - `round() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#round()>`__
+     - :manual:`$round </reference/operator/aggregation/round/>`
+
+   * - `subtract() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlNumber.html#subtract(com.mongodb.client.model.mql.MqlNumber)>`__
+     - :manual:`$subtract </reference/operator/aggregation/subtract/>`
+
+Suppose you need to calculate the total cost of a sale from the item's
+cost, the quantity sold, and an applicable discount.
+
+To calculate the cost of the sale, multiply the ``itemPrice`` and the ``quantity``
+fields by using the ``multiply()`` method. To apply the ``discount``, use the
+``subtract()`` method.
+
+The following code shows the pipeline stage for this aggregation and assigns
+cost of the sale to a new field ``totalSale``:
```

Comment:
S: since the projection is the stage, it would make more sense to explain the projection in the first of these two paragraphs, something like:

```suggestion
To calculate the cost of the sale, multiply the ``itemPrice`` and the ``quantity``
fields by using the ``multiply()`` method. To apply the ``discount``, use the
``subtract()`` method. To output the result as the ``totalSale`` field, use the ``project()`` method.

The following code shows the pipeline stage for this aggregation:
```